homework/index.js

```diff
@@ -44,10 +102,13 @@
         return;
       }
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      repos.sort(sortRepos).forEach(repo => renderRepoDetails(repo, ul));
     });
+    const mainTitle = document.createElement('header');
```

Comment:
Why are you not using `createAndAppend()` here?

Reply:
Well, no answer on this for myself also :) I'll make the change.